config.txt

```diff
@@ -1,9 +1,9 @@
 BASE_IMAGE_REPO=gcr.io/deeplearning-platform-release
-BASE_IMAGE_TAG=m85
+BASE_IMAGE_TAG=m86
 CPU_BASE_IMAGE_NAME=tf2-cpu.2-6
 GPU_BASE_IMAGE_NAME=tf2-gpu.2-6
 LIGHTGBM_VERSION=3.2.1
 TORCH_VERSION=1.9.1
 TORCHAUDIO_VERSION=0.9.1
 TORCHTEXT_VERSION=0.10.1
-TORCHVISION_VERSION=0.10.1
\ No newline at end of file
+TORCHVISION_VERSION=0.10.1
```

Comment:
I don't see any diff?

Reply:
This is just removing the extra space at the end of the line: https://screenshot.googleplex.com/AMCJarhsuDsNoXi

It is a noop